runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -314,8 +314,8 @@ public void onMenuEntryAdded(MenuEntryAdded menuEntryAdded)
 			&& menuEntryAdded.getOption().startsWith("Deposit-"))
 		{
 			ShiftDepositMode shiftDepositMode = config.bankDepositShiftClick();
-			final int actionId = shiftDepositMode.getMenuAction().getId();
-			final int opId = shiftDepositMode.getIdentifier();
+			final int opId = menuEntryAdded.getActionParam1() == 983043 ? shiftDepositMode.getIdentifier() : shiftDepositMode.getIdentifierDepositBox();
```

Comment:
```suggestion
			final int opId = TO_GROUP(menuEntryAdded.getActionParam1()) == BANK_INVENTORY_GROUP_ID ? shiftDepositMode.getIdentifier() : shiftDepositMode.getIdentifierDepositBox();
```

Reply:
It also may be worth inverting this logic so it doesn't break any non-bank cases now where it happens to work, though I am not aware of any.